tests/pipeline/test_factor.py

```diff
@@ -585,6 +586,37 @@ def test_returns(self, seed_value, window_length):
 
         check_allclose(expected, out)
 
+    @parameterized.expand([
+        (100, 15),
+        (101, 4),
+        (102, 100),
+        ])
+    def test_percentchange(self, seed_value, window_length):
+
+        pct_change = PercentChange(
+            inputs=(),
```

Comment:
I think we still probably want a length-1 list of inputs here rather than an empty tuple. Ideally, we'd use `SingleInputMixin` to require exactly one input.

Reply:
Oh right, I forgot about that.  Will make that change now